src/App.tsx

```diff
@@ -15,36 +27,106 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+function getPreparedGoods(
+  goods: string[],
+  { sortField, isReversed }: SortOption,
+): string[] {
+  const prepareGoods = [...goods];
+
+  if (sortField) {
```

Comment:
The condition `if (sortField)` checks if `sortField` is truthy. Since `SortType.None` is an empty string, this condition works as expected. However, it might be clearer to explicitly check for `SortType.None` to improve readability: `if (sortField !== SortType.None)`.